api/src/main/resources/cb_review_api.yaml

```diff
@@ -838,61 +721,111 @@ definitions:
         description: the count for this criteria type in the cohort
 
   ParticipantData:
-    discriminator: dataType
+    discriminator: domainType
     required:
       - itemDate
       - standardVocabulary
       - standardName
       - sourceValue
       - sourceVocabulary
       - sourceName
-      - age
-      - dataType # required for inheritance to work
+      - domainType # required for inheritance to work
     properties:
       itemDate:
         type: string
-        description: The date of the condition occurrence
+        description: The date of the data
       standardVocabulary:
         type: string
-        description: The standard vocabulary of this condition
+        description: The standard vocabulary of this data
       standardName:
         type: string
-        description: The standard name of this condition
+        description: The standard name of this data
       sourceValue:
         type: string
-        description: The source value of this condition
+        description: The source value of this data
       sourceVocabulary:
         type: string
-        description: The source vocabulary of this condition
+        description: The source vocabulary of this data
       sourceName:
         type: string
-        description: The source name of this condition.
-      age:
-        type: integer
-        description: The age of event for the participant.
-      dataType:
-        $ref: "#/definitions/DataType"
+        description: The source name of this data.
+      domainType:
+        $ref: "#/definitions/DomainType"
 
-  ParticipantCondition:
+  Condition:
     allOf:
       - $ref: '#/definitions/ParticipantData'
+      - properties: # extra properties
+          age:
+            type: integer
```

Comment:
See above. You could just add age to ParticipantData and return it everywhere instead?